env/netbox.env

```diff
@@ -20,7 +20,7 @@ REDIS_DATABASE=0
 REDIS_SSL=false
 REDIS_CACHE_HOST=redis-cache
 REDIS_CACHE_PASSWORD=t4Ph722qJ5QHeQ1qfu36
-REDIS_CACHE_DATABASE=0
+REDIS_CACHE_DATABASE=1
```

Comment:
Why `1` if it's a separate server anyway?

Reply:
My thought behind that was, that it would prevent problems if one only changed the Redis host to use the same as the other Redis instance.
Also: In the configuration.py we are already using DB 1 as default, so now the default ist the same in those to places.

Reply:
Makes sense.

Reply:
Although: Is it our configuration.py with this default, or Netbox's?

Reply:
https://github.com/netbox-community/netbox/blob/develop/netbox/netbox/configuration.example.py#L39-L49

```python
REDIS = {
    'webhooks': {
        'HOST': 'localhost',
        'PORT': 6379,
        # Comment out `HOST` and `PORT` lines and uncomment the following if using Redis Sentinel
        # 'SENTINELS': [('mysentinel.redis.example.com', 6379)],
        # 'SENTINEL_SERVICE': 'netbox',
        'PASSWORD': '',
        'DATABASE': 0,
        'DEFAULT_TIMEOUT': 300,
        'SSL': False,
    },
    'caching': {
        'HOST': 'localhost',
        'PORT': 6379,
        # Comment out `HOST` and `PORT` lines and uncomment the following if using Redis Sentinel
        # 'SENTINELS': [('mysentinel.redis.example.com', 6379)],
        # 'SENTINEL_SERVICE': 'netbox',
        'PASSWORD': '',
        'DATABASE': 1,
        'DEFAULT_TIMEOUT': 300,
        'SSL': False,
    }
}
```

Definitely in Netbox as well.

Reply:
Upstream Netbox default is also 1 for the cache DB.